ifrs17/Utils/ImportCalculationMethods.ipynb

```diff
@@ -398,6 +387,33 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Initialize the values given the Period"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static double[] InitializeValues(double value, int period = 0) => Enumerable.Repeat(0.0d, 13).Select((y, i) => i == period ? value : y).ToArray(); "
```

Comment:
13 => period + 1

In this way you don't have arrays longer than necessary. And there is no need for pruning at the end.

Reply:
removed